docs/Quickstart-As-Application.md

```diff
@@ -49,7 +49,7 @@ After the database is initialized we load the data.
 And wait. It will load. If you'd like to watch it load, you can run
 
 ```bash
-./cayley load -c cayley_overview.yml -i data/testdata.nq --alsologtostderr
+./cayley load -c cayley_overview.yml -i data/testdata.nq --logtostderr=true
```

Comment:
It seems that alsologtostderr is still in glog, but the remap wasn't working. I think that the entry for alsologtostderr in [cayley.go:170](https://github.com/cayleygraph/cayley/blob/master/cmd/cayley/cayley.go#L170) was trying to avoid passing the boolean for convenience, but that wasn't working for me.